testutil/simapp/simapp.go

```diff
@@ -32,6 +39,13 @@ func SetupTestingAppConsumer() (ibctesting.TestingApp, map[string]json.RawMessag
 	return testApp, appConsumer.NewDefaultGenesisState(encoding.Marshaler)
 }
 
+func SetupCosmWasmAppConsumer() (ibctesting.TestingApp, map[string]json.RawMessage) {
```

Comment:
Would the `ConsumerApp` interface get rid of this function?

Reply:
No, we must keep separate functions that will instantiate app for the requested consumer chain type. Because SetupTestingAppConsumer() instantiates Minimal type of chain, and SetupCosmWasmAppConsumer() CosmWasm chain.